subworkflows/local/align.nf

```diff
@@ -19,7 +19,7 @@ workflow ALIGN {
         ch_versions   = Channel.empty()
 
         if( aligner == "bwamem2" ) {
-            ALIGN_BWAMEM2 ( reads_input, index )
+            ALIGN_BWAMEM2 ( reads_input, index, fasta, fai )
```

Comment:
Do you have to join reads_input and index maybe?

Reply:
The current version of bwamem2 module expects the items as separate channels now. 😄